sematic/ui/src/loginGoogle.tsx

```diff
@@ -0,0 +1,77 @@
+import Alert from "@mui/material/Alert/Alert";
+import Paper from "@mui/material/Paper/Paper";
+import { CredentialResponse, GoogleLogin, GoogleOAuthProvider } from "@react-oauth/google";
+import { useAtom } from "jotai";
+import { useCallback, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import logo from "./Fox.png";
+import { useAppContext, userAtom } from "./hooks/appHooks";
+import { useHttpClient } from "./hooks/httpHooks";
+import { GoogleLoginPayload } from "./Payloads";
+
+
+export default function GoogleLoginPage() {
```

Comment:
I think there should be a single login page that can feature multiple login buttons depending on what the server returns. It should not be a different page for different login providers.

Reply:
Right, thanks for this feedback. I refactored the file of `login.tsx` a bit, which now includes an overall component and a google-specific component. Would you mind taking another look?